src/Build/Graph/ProjectGraph.cs

```diff
@@ -39,29 +39,40 @@ public sealed class ProjectGraph
         private readonly Dictionary<ConfigurationMetadata, ProjectGraphNode> _allParsedProjects =
             new Dictionary<ConfigurationMetadata, ProjectGraphNode>();
 
-        // TODO: We probably want to mirror all relevant constructor overloads from Project
-
         /// <summary>
-        /// Constructs a graph starting from the given project file.
+        /// Constructs a graph starting from the given project file, evaluating with the global project collection's global properties.
```

Comment:
it's not the project collection's global properties is it? Isn't it just global PC and no specified properties?

Reply:
Oops, that's actually a bug on our end then. `Project` and `ProjectInstance` seem to use the PC's props as a base and applies anything else provided in addition. Currently `ProjectGraph` doesn't use the PC's props at all.

So the documentation says the correct intent, but the code doesn't follow that. Will fix that here.

Reply:
Actually, I'm wrong again. `Project` uses PC as a base, while `ProjectInstance` does not. I'll just correct the comment then and we can think about what that "right" behavior is separately.